<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_charge_send">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>126</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>senden</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>701</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_charge">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>126</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lade Speicher mit maximal</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_max_charge_current">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(aktuell: x W)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_discharge">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>156</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_max_discharge_current">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(aktuell: x W)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Entlade Speicher mit maximal</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_discharge_send">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>156</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>senden</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_percent_min">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>186</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_percent_min_current">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(aktuell: x %)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimaler Ladestand</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min_percent_send">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>186</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>senden</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_chargelimit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ladelimit aktiviert</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_dischargelimit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Entladelimit aktiviert</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>631</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Einstellungen am GEN24: Modbus Port: 502, SunSpec Model: egal, Zähleradresse 200,
Wechselrichtersteuerung über Modbus aktiviert.
Wenn die Steuerung über Modbus wieder deaktiviert wird, dann geht alles in den Ursprungszustand zurück.

Wenn man z.B: mit 500W laden möchte, dann muss man beim Ladelimit 500 eingeben, beim Entladelimit -500 und beide Häkchen setzen zum Aktivieren der Limits.

Fürs entladen mit 500W isses dann genau andersrum

.Wenn der Speicher im Energiesparmodus ist, dann muss er erst aufgeweckt werden indem der minimale Ladestand auf kleiner als der aktuelle Ladezustand gesetzt wird.

Wenn ein minimaler Ladestand nicht angenommen wird, dann kann das unter anderem daran liegen, dass andere Einstellungen am Wechselrichter dies verhindern. Z.B.: minimaler Ladestand in den Speichereinstellungen, Notstromeinstellungen, PV Point Einstellungen.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Adresse</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ip">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>6</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_connect">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>6</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>verbinden</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_disconnect">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>6</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>trennen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_version">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>version</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
